tests/functional/test_snap_dcgm.py

```diff
@@ -0,0 +1,66 @@
+import json
+import subprocess
+import urllib.request
+from time import sleep
+
+from tenacity import retry, stop_after_delay, wait_fixed
+
+
+@retry(wait=wait_fixed(5), stop=stop_after_delay(30))
+def test_dcgm_exporter():
+    """Test of the dcgm-exporter service and its endpoint."""
+    endpoint = "http://localhost:9400/metrics"
+
+    # Check the exporter endpoint, will raise an exception if the endpoint is not reachable
+    urllib.request.urlopen(endpoint)
+
+
+def test_dcgm_nv_hostengine():
```

Comment:
question: why test for the nv-hostengine service here, but test the dcgm-exporter service in conftest?